coa-helm/values.yaml

```diff
@@ -18,16 +18,17 @@ config:
   usePreemptibleVmsOnly: false
   relayListenerConnectionString: RUNTIME_PARAMETER
   relayListenerConnectionName: RUNTIME_PARAMETER
-#  relayListenerSamResourceId:
-#  relayListenerSamUrl:
+  relayListenerSamResourceId: RUNTIME_PARAMETER
+  relayListenerSamResourceType: "workspace"
+  relayListenerSamUrl: RUNTIME_PARAMETER
   relayListenerTargetHost: "http://cromwell:8000/"
 
 images:
   mysql: mysql:latest
   cromwell: broadinstitute/cromwell:79
   tes: mcr.microsoft.com/cromwellonazure/tes:2
   triggerservice: mcr.microsoft.com/cromwellonazure/triggerservice:2
-  relaylistener: terradevacrpublic.azurecr.io/terra-azure-relay-listeners:b26c311
+  relaylistener: terradevacrpublic.azurecr.io/terra-azure-relay-listeners:bb18347 # https://github.com/DataBiosphere/terra-azure-relay-listeners/pull/17

```

Comment:
Very handy that the listener repo automatically builds and pushes an image for each PR.